<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalStripEditorWindow</class>
 <widget class="QMainWindow" name="TerminalStripEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de borniers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="m_stacked_widget"/>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="m_button_box">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_add_terminal_strip"/>
   <addaction name="m_remove_terminal"/>
   <addaction name="m_reload"/>
  </widget>
  <action name="m_add_terminal_strip">
   <property name="icon">
    <iconset resource="../../../qelectrotech.qrc">
     <normaloff>:/ico/16x16/list-add.png</normaloff>:/ico/16x16/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Ajouter un bornier</string>
   </property>
   <property name="toolTip">
    <string>Ajouter un bornier au projet</string>
   </property>
  </action>
  <action name="m_remove_terminal">
   <property name="icon">
    <iconset resource="../../../qelectrotech.qrc">
     <normaloff>:/ico/16x16/list-remove.png</normaloff>:/ico/16x16/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer le bornier</string>
   </property>
   <property name="toolTip">
    <string>Supprimer le bornier du projet</string>
   </property>
  </action>
  <action name="m_reload">
   <property name="icon">
    <iconset resource="../../../qelectrotech.qrc">
     <normaloff>:/ico/16x16/view-refresh.png</normaloff>:/ico/16x16/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Recharger</string>
   </property>
   <property name="toolTip">
    <string>Recharger les borniers</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../qelectrotech.qrc"/>
 </resources>
 <connections/>
</ui>
